### PR TITLE
[#186820573] Add summaryDescriptionMd field to Quick Action Tasks

### PR DESCRIPTION
**Pull request by seidior**
## Description

This adds a `summaryDescriptionMd` field to Quick Action Tasks, similar to our other pieces of content. Also included with this pull request:

- Migration of content that makes sense into the new `summaryDescriptionMd` field.
  - Modified the content by adding line breaks so it would render in the CMS preview properly.
- Made the new `summaryDescriptionMd` field for this content searchable.
- Fixes to the content so it has the proper heading order (h2 -> h3 -> h4).

### Ticket

[#186820573](https://www.pivotaltracker.com/story/show/186820573)

### Approach

Nothing new here. Very similar task to the headings updates.

### Steps to Test

Verify each piece of content in the CMS to ensure it renders properly. Not sure if each content item can be triggered to display manually within the app.

### Notes

Will need verification from content that some of these Quick Action Tasks are ineligible for the `summaryDescriptionMd` treatment. Further, if any of these have been modified in `content-repo`, we'll have some merge conflict issues.

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation (EX: [Engineering Reference/FAQ](https://docs.google.com/document/d/1X4iWSGmBZdHYZ0jGqkwTR3_yyyqI_TiiJptfc8pi9Po)), if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config
- [x] If I added/updated any values in `userData` (including `profileData`, `formationData` etc), then I added a new migration file
- [x] If I added any new collections to the CMS config, then I updated the search tool and `cmsCollections.ts`

### CODE REVIEW

File: web/src/pages/actions/[quickActionTaskUrlSlug].tsx [hunk 1]
```diff
@@ -33,6 +33,12 @@ export const QuickActionElement = (props: Props): ReactElement => {
           <h1>{props.quickActionTask.name}</h1>
         </div>
       </div>
+      {props.quickActionTask.summaryDescriptionMd?.length > 0 && (
```
Comment (FarazA22) [inline web/src/pages/actions/[quickActionTaskUrlSlug].tsx:36]: dust, we don't need `?.length > 0` right? It would not render if there is an empty string.

Comment (seidior) [reply web/src/pages/actions/[quickActionTaskUrlSlug].tsx:36]: @FarazA22 We otherwise don't want to render the `<HorizontalRule />`, right?

Comment (FarazA22) [reply web/src/pages/actions/[quickActionTaskUrlSlug].tsx:36]: Yeah I think the HR would only render if the summary text renders.

File: content/src/quick-action-tasks/electrical-contracting-business-permit-update.md [hunk 2]
```diff
@@ -1,49 +1,56 @@
...
-:::
+
+  :::infoAlert
+
+  It is recommended to apply at least ten days **before** the next scheduled monthly meeting of the Board of Examiners of Electrical Contractors. Dates are posted on the [Meetings](https://www.njconsumeraffairs.gov/elec/Pages/meetings.aspx) page.
+
+  :::
+---
 
 ### Application Requirements
```
Comment (AnJuHyppolite) [inline content/src/quick-action-tasks/electrical-contracting-business-permit-update.md:29]: [pebble] Should this be a third-level heading as denoted here (`###`) or second-level heading (`##`)? If it's the first and only heading, I noticed our pattern is second-level.

File: web/src/pages/actions/[quickActionTaskUrlSlug].tsx (hunk 1 above)
Comment (AnJuHyppolite) [reply web/src/pages/actions/[quickActionTaskUrlSlug].tsx:36]: I think I agree with Faraz.

File: content/src/quick-action-tasks/electrical-contracting-business-permit-update.md (hunk 2 above)
Comment (seidior) [reply content/src/quick-action-tasks/electrical-contracting-business-permit-update.md:29]: Good catch.

**Pull request merged**